Client/metrics.yaml

```diff
@@ -525,6 +525,47 @@ qr_code:
       - firefox-ios@mozilla.com
     expires: "2021-07-01"
 
+# Default browser card metrics
+default_browser_card:
+  dismiss_pressed:
+    type: counter
+    description: |
+      Counts the number of times default browser card is dismissed.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1644846
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1644846
+    notification_emails:
+      - firefox-ios@mozilla.com
+    expires: "2021-07-01"
+  go_to_settings_pressed:
+    type: counter
+    description: |
+      Counts the number of times the Go To Settings button on
+      default browser card is clicked.
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/7151
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/7245
+    notification_emails:
+      - firefox-ios@mozilla.com
+    expires: "2021-03-01"
+
+app:
+  opened_as_default_browser:
+    type: counter
+    description: |
+      Recorded when a preference is changed and includes the
+      preference that changed as well as the value changed to
+      recorded in the extra keys.
```

Comment:
This sounds like it could be a labeled counter https://mozilla.github.io/glean/book/user/metrics/labeled_counters.html

If you use extras, do you need `has_extras`? Also include what all possible values are (though I guess it could just be enabled/disabled)

Reply:
There is only one value for this event so I am not sure a labeled counter is appropriate. It is only getting called when it is for sure opened as a default browser; it won't be true/false.